test/init.js

```diff
@@ -32,6 +32,7 @@ const skips = [
   'find all skip two hundred',
   'isActual',
   'with an invalid connection',
+  'cloudant geo',
```

Comment:
The test is skipped because geo query not working as before.
I updated the test case according to https://github.com/cloudant/nodejs-cloudant#cloudant-geospatial, and realized the example doesn't return the expected result. The query is rejected with "Document is missing" error.
Gist https://gist.github.com/jannyHou/f865a1cb9108c3848e63f97177f58030#file-gistfile1-js-L61 reproduces it, confirming with cloudant team on slack, will create an issue in https://github.com/cloudant/nodejs-cloudant if it's a bug.